docs/tutorials/clojure-guide.md

```diff
@@ -0,0 +1,225 @@
+---
+author: ericdallo
+disqus: emacs-lsp
+---
+
+# Configuring `Emacs` as a `Clojure` IDE
+
+In this guide, I will show you how to configure emacs to develop for Clojure
+using: 
+
+- `lsp-mode`, which have a lot of IDE features like **completion**, **renaming**
+**find-references**, **find-definition**, **code lens**, **code actions**
+- [cider](https://github.com/clojure-emacs/cider),
+the Emacs package most famous for a Clojure **REPL driven development**. 
+
+## Why should I try LSP for Clojure?
+
+When I started programming for Clojure in Emacs coming from Java world, it was odd to me that 
+for almost everything you need to do, **you need a REPL running in your project**, and if you change anything
+on it, you would need to reload the namespace/code that was changed to apply changes on your REPL, also I 
+missed a lot of features I used to use on my day that works pretty well for Java like cleaning unused imports, 
+call hierarchy of a method/function, formatting, real-time syntax checking, and code actions like adding a missing import.
+Then I found [clojure-lsp](https://github.com/clojure-lsp/clojure-lsp), which implements most of those features and 
+since that, I use it along with `cider` for more than a year without issues. 
```

Comment:
``` suggestion
Coming from the Java world, when I started programming for Clojure in Emacs, it was odd to me that 
for almost everything you needed to do with Clojure, **you needed a REPL running in your project**, and if you changed anything
in your project, you would need to reload the modified namespace/code to see the changes in the REPL.
I also
missed a lot of features popularized by Java IDEs, like cleaning unused imports, 
viewing the call hierarchy of a method/function, formatting code automatically, checking syntax in real-time, and using code actions that could, for example, add a missing import.
Ever since I found [clojure-lsp](https://github.com/clojure-lsp/clojure-lsp), which implements most of those features, I have been using it alongside `cider` without issues.
```